svm-decode/src/lib.rs

```diff
@@ -136,9 +154,9 @@ pub fn decode_instructions(
     Ok(batch)
 }
 
-pub fn match_discriminators(instr_data: &[u8], discriminator: &[u8]) -> Result<Vec<u8>> {
+pub fn match_discriminators(instr_data: &[u8], discriminator: &Vec<u8>) -> Result<Vec<u8>> {
```

Comment:
to_vec should work the same with passing slice so don't need to change it to Vec<u8>

Reply:
Clippy complains about that. I left your way.